quiz.js

```diff
@@ -0,0 +1,58 @@
+
+var tree = {
+	height: document.getElementById("height"),
+	character: document.getElementById("character")
+};
+
+var submitButton = document.getElementById("submit");
+var clearButton = document.getElementById("clear");
+var output = document.getElementById("treeDOM");
+
+
+function growYourTree () {
+	var howTall = tree.height.value;
```

Comment:
before this line add in:
e.preventDefault()

this keeps the form submission from trying to change pages - so the tree will stay in the console.